torch/_inductor/config.py

```diff
@@ -1233,6 +1233,12 @@ class triton:
     # instead of recording and executing cudagraphs
     force_cudagraphs_warmup = False
 
+    # If False (default), torch.compile skips cudagraph for a graph if it
+    # contains cudagraph-unsafe ops. If True, we require that all cuda ops
+    # be captured into cudagraph. If this is not possible, this will raise
+    # an error.
+    cudagraph_or_error = os.environ.get("TORCHINDUCTOR_CUDAGRAPH_OR_ERROR", 0) == "1"
```

Comment:
Can you use the fancy `from torch.utils._config_module import Config` API instead? Thanks.